contrib/dex/README.md

```diff
@@ -0,0 +1,65 @@
+## Exposing DEX service
+After the cluster is up, you have to manually expose dex service using a ELB or Ingress.
```

Comment:
Could `dex` go under the `addons` top level config I added during the rescheduler work?